<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoublePolarDlg</class>
 <widget class="QDialog" name="DoublePolarDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Double Polar Calculation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Station 1</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="NameLineEdit" name="from1Edit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Setup</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="IntLineEdit" name="setup1Edit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Target</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="NameLineEdit" name="trg1Edit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Direction</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="AngleLineEdit" name="dir1Edit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Correction</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="AngleLineEdit" name="oc1Edit"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Vertical</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="AngleLineEdit" name="ver1Edit"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Slope</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="LengthLineEdit" name="dis1Edit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Station 2</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="NameLineEdit" name="from2Edit"/>
        </item>
        <item row="1" column="1">
         <widget class="IntLineEdit" name="setup2Edit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Setup</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="NameLineEdit" name="trg2Edit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Target</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="AngleLineEdit" name="dir2Edit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Direction</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="AngleLineEdit" name="oc2Edit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Correction</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="AngleLineEdit" name="ver2Edit"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Vertical</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="LengthLineEdit" name="dis2Edit"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Slope</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="calculateButton">
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Result</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="meanRadio">
          <property name="text">
           <string>Mean</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="station1Radio">
          <property name="text">
           <string>Station 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="station2Radio">
          <property name="text">
           <string>Station 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="weightedRadio">
          <property name="text">
           <string>Weighted mean</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Target</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="NameLineEdit" name="targetEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Easting</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DoubleLineEdit" name="eastingEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Northing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DoubleLineEdit" name="northingEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NameLineEdit</class>
   <extends>QLineEdit</extends>
   <header>customcontrols.h</header>
  </customwidget>
  <customwidget>
   <class>IntLineEdit</class>
   <extends>QLineEdit</extends>
   <header>customcontrols.h</header>
  </customwidget>
  <customwidget>
   <class>AngleLineEdit</class>
   <extends>QLineEdit</extends>
   <header>customcontrols.h</header>
  </customwidget>
  <customwidget>
   <class>LengthLineEdit</class>
   <extends>QLineEdit</extends>
   <header>customcontrols.h</header>
  </customwidget>
  <customwidget>
   <class>DoubleLineEdit</class>
   <extends>QLineEdit</extends>
   <header>customcontrols.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
